app/views/accounts/_form.html.erb

```diff
@@ -42,6 +42,17 @@
     <% end %>
   </div>
 
+  <div class="field">
+    <%= form.label :district_id, "District" %>
+    <% if @districts %>  
+      <%= form.select(:district_id) do %>
+        <% @districts.each do |c| -%>
```

Comment:
Why this value have name `c`? If you itterate districts name should be `district` or `d`